engine/lib/access.php

```diff
@@ -594,6 +595,108 @@ function elgg_override_permissions($hook, $type, $value, $params) {
 }
 
 /**
+ * Creates a Friends ACL for a user
+ *
+ * @param \Elgg\Event $event
```

Comment:
Could you document the event here. I think it's `@elgg_event`. Makes it easier to read and know what event this handler is for